src/main/scala/org/allenai/ike/BlackLabResult.scala

```diff
@@ -10,7 +9,8 @@ case class BlackLabResult(
     wordData: Seq[WordData],
     matchOffset: Interval,
     captureGroups: Map[String, Interval],
-    corpusName: String
+    corpusName: String,
+    sourceDocument: Option[Int] = None
```

Comment:
So, I was looking through the code changes and Blacklab's API, and this document id is part of the returned hit object, so would it be better to just make another case class that includes this information? I am struggling with how to pull this information out of Blacklab outside of when the result is created.